### PR TITLE
[W6.1e][T16-A2] Ng Shi Kang

### PR DESCRIPTION
**Pull request by ngshikang**
Add JUnit test for isAnyNull method

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -43,4 +43,26 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull(){
```
Comment (ngshikang) [inline test/java/seedu/addressbook/common/UtilsTest.java:48]: isAnyNull_containsNull_returnsTrue()

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -43,4 +43,26 @@ private void assertAreUnique(Object... objects) {
...
+        // no objects
+        assertNoneNull();
+        // contains null object
```
Comment (ngshikang) [inline test/java/seedu/addressbook/common/UtilsTest.java:51]: can be extracted into a separate unit test

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 3]
```diff
@@ -43,4 +43,26 @@ private void assertAreUnique(Object... objects) {
...
+        assertIsAnyNull(null, "a", "b", null);
+        assertIsAnyNull(null, 1, new Integer(1));
+        //no null objects
+        assertNoneNull(1);
+        assertNoneNull("");
+        assertNoneNull("abc");
+        assertNoneNull("abc", "abc");
+        assertNoneNull(1, new Integer(1));
+        assertNoneNull("abc", "", "abc", "ABC");
+        assertNoneNull("", "abc", "a", "abc");
```
Comment (asiddharth) [inline test/java/seedu/addressbook/common/UtilsTest.java:64]: Should we also add a test case to check if a list containing a Null object returns false?

**Review by asiddharth — COMMENTED** [review]
@ngshikang Good Job!
